koku/api/query_params.py

```diff
@@ -181,6 +185,36 @@ def _process_tag_query_params(self, query_params):
                 param_tag_keys.add(key)
         return param_tag_keys
 
+    # TODO: This function will be needed for below
+    # def _strip_aws_category_prefix(self, value):
+    #     if AWS_CATEGORY_PREFIX.replace(":", "") not in value:
+    #         return value
+    #     if value.startswith(AWS_CATEGORY_PREFIX):
+    #         return value[len(AWS_CATEGORY_PREFIX) :]  # noqa: E203
+    #     if value.startswith(AND_AWS_CATEGORY_PREFIX):
+    #         return value[len(AND_AWS_CATEGORY_PREFIX) :]  # noqa: E203
+    #     if value.startswith(OR_AWS_CATEGORY_PREFIX):
+    #         return value[len(OR_AWS_CATEGORY_PREFIX) :]  # noqa: E203
```

Comment:
Quote from above:
```
It is not going to completed during this issue, I talked to Andrew this is an MVP and for some internal stakeholders. We are going to build the right urls for a customer and add them to an email we control as a quickest win scenario. Then we will go back and create a table that has the acceptable categories since the categories are created by the user it will work very similarly to what we do for tags in the reporting_awstags_summary table. However, that will be in a separate PR, and once we have that we can figure out if a category is acceptable as a parameter. So, these todos are to make it easier for when we come back to it later.
```